google-cloud-translate-v2/lib/google/cloud/translate/v2/service.rb

```diff
@@ -118,7 +118,9 @@ def default_http_headers
                 "Content-Type"                 => "application/json",
                 "x-goog-api-client"            => "gl-ruby/#{RUBY_VERSION} gccl/#{lib_version}"
               }
-              headers["x-goog-user-project"] = credentials.quota_project_id if credentials.respond_to? :quota_project_id
+              quota_project = V2.translation_config&.quota_project
```

Comment:
Where is `quota_project` defined for the `translation_config` Config?

Reply:
It's in the `google-cloud-translate` wrapper. If the wrapper is present, `translation_config` is non-nil. This is different from the other versioned libraries which generally have their own "override" configs that are always available even if no wrapper/veneer is present. However, Translate V2 is a bizarre special case because it's the only handwritten versioned library. And I've punted on moving it to feature parity with the generated libraries because it's an older version. So this PR's focus is on completing the requirement (`quota_project` works) for the expected normal case (people are installing `google-cloud-translate` instead of `google-cloud-translate-v2`, and hence the configs defined in the wrapper are available.)